src/component/Breadnav.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
```

Comment:
This navigation seems redundant as we currently have 4 ways to get back to the home page. It is useful as a header for the topic we are viewing but we could include this in the articles view component.